### PR TITLE
[SCSE-58] Footer Map Component with Footer Content

### PR DESCRIPTION
**Pull request by venusnmj**
- for 'content.button' type of footer content, edit 'href', 'ctaTitle', 'btnText'
- for 'content.type' type of footer content, edit 'alerts' array

[SCSE-58]

### CODE REVIEW

File: packages/ui/components/footer/Update.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Heading, Flex, Text } from "@chakra-ui/react";
+import React from "react";
+
+export interface UpdateProps {
+    title: string
+    description: string
+}
+
+export const Update = ({ title, description } : UpdateProps) => {
```
Comment (xJQx) [inline packages/ui/components/footer/Update.tsx:9]: Is there a better way to name this component other than "Update"? This name might be abit confusing 🤔

File: packages/ui/components/footer/ContentButton.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import { VStack, Heading } from '@chakra-ui/react';
+import { Button } from 'ui';
+
+export interface ContentButtonProps {
+    href: string
+    title?: string
+    label: string
+}
+
+export const ContentButton = ( { href, title, label } : ContentButtonProps) => {
```
Comment (xJQx) [inline packages/ui/components/footer/ContentButton.tsx:11]: name as 'FooterContentButton"?

File: packages/ui/components/footer/ContentText.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Box } from '@chakra-ui/react';
+import { Update, UpdateProps } from './Update';
+
+export interface ContentTextProps {
+    alerts: Array<UpdateProps>
+}
+
+export const ContentText = ( { alerts } : ContentTextProps) => {
```
Comment (xJQx) [inline packages/ui/components/footer/ContentText.tsx:9]: name as 'FooterContentText"?

**Review by xJQx — CHANGES_REQUESTED** [review]

**Comment by venusnmj** [comment]
work in progress to create separate storybook components for footer content button and footer content text components

**Review by xJQx — CHANGES_REQUESTED** [review]

**Review by xJQx — APPROVED** [review]
lgtm 🚀

**Pull request merged**